modules/services/dunst.nix

```diff
@@ -90,7 +90,7 @@ in
         xdg.dataFile."dbus-1/services/org.knopwob.dunst.service".source =
           "${pkgs.dunst}/share/dbus-1/services/org.knopwob.dunst.service";
 
-        services.dunst.settings.global.icon_folders =
+        services.dunst.settings.global.icon_path =
```

Comment:
which version of dunst was this deprecated? is the version in 18.09 has it already deprecated? I’m thinking we might have to compute the attribute name conditionally. You can achieve that by computing conditionally around dunst version (there’s a helper for that but the name slippe dmy mind, maybe lib.versionOf) in a let statement.

Reply:
I'm running 18.09, and it has dunst-1.3.2. Option «icon_folders» causes deprecation warning to be logged. It still works though.

Reply:
I can confirm that 18.09 is running v1.3.2 so this should be a safe change. LGTM!